project/ValidatePullRequest.scala

```diff
@@ -4,189 +4,42 @@
-        case (None, Some(branch)) => s"origin/$branch" // usually would be "master" or "release-2.3" etc
-        case (None, None)         => "origin/master" // defaulting to diffing with "master"
-      }
-    },
-
-    buildAllKeyword in Global in ValidatePR := """PLS BUILD ALL""".r,
-
-    githubEnforcedBuildAll in Global in ValidatePR := {
-      val log = streams.value.log
-      val buildAllMagicPhrase = (buildAllKeyword in ValidatePR).value
-
-      sys.env.get(PullIdEnvVarName).map(_.toInt) flatMap { prId =>
-        log.info("Checking GitHub comments for PR validation options...")
-
-        try {
-          import scala.collection.JavaConverters._
-          val gh = GitHubBuilder.fromEnvironment().withOAuthToken(GitHub.envTokenOrThrow).build()
-          val comments = gh.getRepository("akka/akka").getIssue(prId).getComments.asScala
-
-          def triggersBuildAll(c: GHIssueComment): Boolean = buildAllMagicPhrase.findFirstIn(c.getBody).isDefined
-          comments collectFirst { case c if triggersBuildAll(c) =>
-            BuildCommentForcedAll(buildAllMagicPhrase.toString(), c)
-          }
-        } catch {
-          case ex: Exception =>
-            log.warn("Unable to reach GitHub! Exception was: " + ex.getMessage)
-            None
-        }
-      }
-    },
-
-    changedDirectories in Global in ValidatePR := {
-      val log = streams.value.log
-
-      val prId = (sourceBranch in ValidatePR).value
-
-      val target = (targetBranch in ValidatePR).value
-
-      // TODO could use jgit
-      log.info(s"Diffing [$prId] to determine changed modules in PR...")
-      val diffOutput = s"git diff $target --name-only".!!.split("\n")
-      val diffedModuleNames =
-        diffOutput
-          .map(l => l.trim)
-          .filter(l =>
-            l.startsWith("akka-") ||
-            (l.startsWith("project") && l != "project/MiMa.scala")
-          )
-          .map(l ⇒ l.takeWhile(_ != '/'))
-          .toSet
-
-      val dirtyModuleNames: Set[String] =
-        if (runningOnJenkins) Set.empty
-        else {
-          val statusOutput = s"git status --short".!!.split("\n")
-          val dirtyDirectories = statusOutput
-            .map(l ⇒ l.trim.dropWhile(_ != ' ').drop(1))
-            .map(_.takeWhile(_ != '/'))
-            .filter(dir => dir.startsWith("akka-") || dir == "project")
-            .toSet
-          log.info("Detected uncommitted changes in directories (including in dependency analysis): " + dirtyDirectories.mkString("[", ",", "]"))
-          dirtyDirectories
-        }
-
-
-      val allModuleNames = dirtyModuleNames ++ diffedModuleNames
-      log.info("Detected changes in directories: " + allModuleNames.mkString("[", ", ", "]"))
-      allModuleNames
-    }
+    validatePullRequest / includeFilter := PathGlobFilter("akka-*/**"),
+    validatePullRequestBuildAll / excludeFilter := PathGlobFilter("project/MiMa.scala"),
+    prValidatorGithubRepository := Some("akka/akka")
```

Comment:
missing `ThisBuild / prValidatorBuildAllKeyword := "PLS BUILD ALL".r`

Reply:
Ah it is by default; the docs on the plugin made the impression it's the opposite. Should be good then